### PR TITLE
Support perl 5.34 and Fedora

### PR DESCRIPTION
**Pull request by michal-josef-spacek**

### CODE REVIEW

**Comment by michal-josef-spacek** [comment]
[test]

**Comment by michal-josef-spacek** [comment]
[test-all]

**Review by phracek — CHANGES_REQUESTED** [review]
Thanks for adding new version, but please follow up how to add a newer version https://github.com/sclorg/welcome/blob/master/contribution.md#adding-a-new-version

**Comment by michal-josef-spacek** [comment]
@phracek ha, thank you for doc.

**Comment by michal-josef-spacek** [comment]
[test]

**Comment by michal-josef-spacek** [comment]
I updated PR to version with 5.34/Dockerfile.c9s on the place and excluded it in the build/test

**Comment by michal-josef-spacek** [comment]
[test]

**Comment by michal-josef-spacek** [comment]
[test-all]

File: 5.34/README.md [hunk 1]
```diff
@@ -69,10 +36,10 @@ To use the Perl image in a Dockerfile, follow these steps:
 #### 1. Pull a base builder image to build on
 
 ```
-podman pull ubi8/perl-532
```
Comment (phracek) [inline 5.34/README.md]: This should remain as ubi8. The README.md file is copied into Red Hat Catalog. Please keep there ubi8. In the whole document.

**Review by phracek — CHANGES_REQUESTED** [review]

File: 5.34/README.md (hunk 1 above)
Comment (michal-josef-spacek) [reply 5.34/README.md]: @phracek I moved to original version this lines.

**Review by phracek — CHANGES_REQUESTED** [review]
One last issue. And then we can merge it.

File: 5.34/README.md [hunk 2]
```diff
@@ -25,37 +23,6 @@ This container image includes an cpanm utility, so users can use it to install P
 modules for their web applications. There is no guarantee for any specific CPAN module
 version, that is included in the image; those versions can be changed anytime.
 
-Usage in Openshift
```
[the remark this answers is not in the document]
Comment (michal-josef-spacek) [reply 5.34/README.md]: I updated README to better version.

**Comment by michal-josef-spacek** [comment]
[test-all]

**Review by phracek — APPROVED** [review]
LGTM. Thanks. Let's get merge it.

**Pull request merged**